client/scss/widgets/_votes-vs-seats.scss

```diff
@@ -0,0 +1,75 @@
+$vvs-bar-height: 10;
```

Comment:
we already have a `components` directory for this, so this should be `client/scss/components/_votes-vs-seats.scss`